setup.py

```diff
@@ -26,9 +29,12 @@ def read_list(fname):
     """
     Load content of the file and split it into a list of lines.
 
-    :param fname: file name (or path relative to the project root)
-    :return: file content as a list of strings (one string per line)
-             with end of lines characters stripped off
+    Args:
+        fname (str): file name (or path relative to the project root)
+
+    Returns:
+        List[str]: file content (one string per line) with end of lines characters stripped off
```

Comment:
Too long line, can we make it less than 80?

Reply:
Sure. I'll also add setup.py to the list of paths to lint (it currently isn't).

Reply:
done that + added `make lint` to CI config